### DISCUSSION TITLE
Client syncing

### DISCUSSION THREAD

**Discussion by IansGithubAcc** [immich-app/immich#3157] [General]
I am new to Immich but am very impressed by it so far. I was wondering if there is a recommended method for data syncing between multiple clients. I'm a big fan of decentralizing my data hence the question. I could find surprisingly little on the matter.

**Comment by alextran1502** [comment]
You can use the CLI to bulk import your data in different machines/VM https://immich.app/docs/features/bulk-upload

**Reply by IansGithubAcc** [reply]
Thank you for the rapid response! Yes I saw the CLI bulk import which could definitely be used for this. Might it be a good idea to have a daily sync option in the CLI? I guess I could write my own script for this purpose, but I can imagen this might be a great feature for many users.

**Reply by etnoy** [reply]
There's some new functionality in the pipe, see #3124

**Reply by IansGithubAcc** [reply]
Very cool! Thanks for sharing! I'll close the thread.
